lib/logger/lib/logger.ex

```diff
@@ -133,15 +133,16 @@ defmodule Logger do
       `:sync_threshold` messages in its queue, `Logger` will change
       to *sync mode*, to apply backpressure to the clients.
       `Logger` will return to *async mode* once the number of messages
-      in the queue is reduced to `sync_threshold * 0.75` messages.
-      Defaults to 20 messages. `:sync_threshold` can be set to `0` to force *sync mode*.
+      in the queue is reduced to one below the `sync_threshold`.
+      Defaults to 20 messages. `:sync_threshold` can be set to `0` to
+      force *sync mode*.
 
     * `:discard_threshold` - if the `Logger` manager has more than
       `:discard_threshold` messages in its queue, `Logger` will change
       to *discard mode* and messages will be discarded directly in the
       clients. `Logger` will return to *sync mode* once the number of
-      messages in the queue is reduced to `discard_threshold * 0.75`
-      messages. Defaults to 500 messages.
+      messages in the queue is reduced to one below `discard_threshold`.
```

Comment:
The new comment above on https://github.com/elixir-lang/elixir/pull/8567/files#diff-e595abf4d143c9f920f1d59b61df7c35R136 uses `one below the` so suggesting adding `the` here to match

```suggestion
      messages in the queue is reduced to one below the `discard_threshold`.
```